source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+=======================
+Halloween Treasure Hunt
+=======================
+
+.. contents:: Table of Contents
+    :local:
+
+Introduction
+============
+
+Welcome back to the **Denizen-for-Bukkit Docs**. This is our first example project, so we'll start with the basics. 
+This guide will cover the following topics:
+
+* Denizen objects and general syntax.
+* Browsing through the documentation.
+* Reloading scripts and debug modes.
+* World script containers, events and context tags.
+* Common script commands and arguments.
+* Constructing tags with simple bases and modifiers.
+* First contact with foreach loops.
+
+If you are not familiar with any of the concepts listed, we suggest revisiting the following tutorial documents:
+
+* :doc:`Section 1.3 - dScript Format </docs/getting-started/dscript-format>`
+* :doc:`Section 2.1 - An Introduction to Tags </docs/basics-of-scripting/an-introduction-to-tags>`
+* :doc:`Section 2.2 - The Basics of Denizen Commands </docs/basics-of-scripting/basics-of-denizen-commands>`
+* :doc:`Section 2.7 - Your First World Script </docs/basics-of-scripting/your-first-world-script>`
+* foreach loops here
```

Comment:
It's probably worth making it more obvious that it's an incomplete list. A todo note, like `**[TODO: Add section reference]**` or similar may be better.